locales/en-US/experiments.ftl

```diff
@@ -252,7 +252,5 @@ voicefillContributors4Title = Firefox UX
 voicefillContributors5Title = Softvision QA
 voicefillContributors6Title = Softvision QA
 voicefillContributors7Title = Advanced Development, Emerging Technologies
-testpilotNewsupdatesGeneralupdate3Title = New experiments launched
-testpilotNewsupdatesGeneralupdate3Content = Introducing Send, Notes, and Voice Fill. Click on the cards below to try them out.
-testpilotNewsupdatesGeneralupdate4Title = We started a blog
-testpilotNewsupdatesGeneralupdate4Content = Check out our brand new blog for in-depth discussion about Firefox Test Pilot.
\ No newline at end of file
+sendNewsupdatesSendupdate1Title = Send with Safari and Edge!
```

Comment:
FWIW, we need to wait to get some l10n review whenever changing strings, so this shouldn't be a quick merge